views-turbo/src/main/java/io/micronaut/views/turbo/DefaultTurboStreamRenderer.java

```diff
@@ -28,12 +26,12 @@
 
 /**
  * {@link io.micronaut.context.annotation.DefaultImplementation} of {@link TurboStreamRenderer}.
+ * @param <R> The request type
  * @author Sergio del Amo
  * @since 3.3.0
  */
 @Singleton
-@Requires(classes = HttpRequest.class)
-public class DefaultTurboStreamRenderer implements TurboStreamRenderer {
+public class DefaultTurboStreamRenderer<R> implements TurboStreamRenderer<R> {
```

Comment:
make it `@Internal`, 
```suggestion
final class DefaultTurboStreamRenderer<R> implements TurboStreamRenderer<R> {
```

Reply:
Really?  It has a protected field 🤔

Reply:
Ok, so marked internal and added final (and made the field private)